.github/renovate.json5

```diff
@@ -1,231 +1,337 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:recommended",
-    "docker:pinDigests",
-    "helpers:pinGitHubActionDigests"
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:recommended',
+    'docker:pinDigests',
+    'helpers:pinGitHubActionDigests',
+  ],
+  ignorePaths: [
+    'instrumentation/**',
   ],
-  "ignorePaths": ["instrumentation/**"],
   // needed in order to get patch-only updates in package rules below
   // unfortunately you can't combine updateTypes and separateMinorPatch in the same package rule
   // so we have to apply it globally here, see
   // https://github.com/renovatebot/renovate/discussions/8399#discussioncomment-305798
-  "separateMinorPatch": true,
-  "packageRules": [
+  separateMinorPatch: true,
+  packageRules: [
     {
       // this is to reduce the number of renovate PRs
-      "matchManagers": [
-        "github-actions",
-        "dockerfile"
+      matchManagers: [
+        'github-actions',
+        'dockerfile',
+      ],
+      extends: [
+        'schedule:weekly',
       ],
-      "extends": ["schedule:weekly"],
-      "groupName": "weekly update",
-      "separateMinorPatch": false // overrides separateMinorPatch specified above
+      groupName: 'weekly update',
+      separateMinorPatch: false,  // overrides separateMinorPatch specified above
     },
     {
-      "matchPackageNames": [
-        "io.opentelemetry.contrib:opentelemetry-aws-resources",
-        "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator",
-        "io.opentelemetry.contrib:opentelemetry-gcp-resources",
-        "io.opentelemetry.contrib:opentelemetry-baggage-processor",
-        "io.opentelemetry.proto:opentelemetry-proto",
-        "io.opentelemetry.semconv:opentelemetry-semconv"
+      matchPackageNames: [
+        'io.opentelemetry.contrib:opentelemetry-aws-resources',
+        'io.opentelemetry.contrib:opentelemetry-aws-xray-propagator',
+        'io.opentelemetry.contrib:opentelemetry-gcp-resources',
+        'io.opentelemetry.contrib:opentelemetry-baggage-processor',
+        'io.opentelemetry.proto:opentelemetry-proto',
+        'io.opentelemetry.semconv:opentelemetry-semconv',
       ],
       // Renovate's default behavior is only to update from unstable -> unstable if it's for the
       // major.minor.patch, under the assumption that you would want to update to the stable version
       // of that release instead of the unstable version for a future release
       // (TODO remove once the artifacts above release stable versions)
-      "ignoreUnstable": false,
-      "allowedVersions": "!/\\-SNAPSHOT$/"
+      ignoreUnstable: false,
+      allowedVersions: '!/\\-SNAPSHOT$/',
     },
     {
-      "matchPackagePrefixes": ["io.quarkus"],
-      "groupName": "quarkus packages"
+      groupName: 'quarkus packages',
+      matchPackageNames: [
+        'io.quarkus{/,}**',
```

Comment:
it seems like we could simplify this to `io.quarkus**`, but I'd suggest we merge as-is and I might try this in a separate PR so that if it doesn't work we have a simple revert